index.html

```diff
@@ -8,41 +9,343 @@
+                    you like what you see and have a project you need coded,
                     don’t hestiate to contact me.<br>
-                    <button type="button" class="connect">Lets Connect!
-                    </button>
-                </p>     
+                    <!-- <button type="button" class="connect">Lets Connect!
+                    </button> -->
+                </p>
                 <ul class="social-media-icons">
-                    <li class="social-media"><a href="https://twitter.com/" ><img src="./icons/Social_icon1.png"  alt="Twitter" ></a></li> 
-                    <li class="social-media"><a href="https://www.linkedin.com/" ><img src="./icons/Social_icon2.png"     alt="Linkedin" ></a></li> 
-                    <li class="social-media"><a  href="https://medium.com/" ><img src="./icons/Social_icon3.png"      alt="Medium" ></a></li> 
-                    <li class="social-media"><a href="https://angel.co/" ><img src="./icons/Social_icon5.png"      alt="angellist"></a></li>
-              </ul>          
+                    <li class="social-media"><a href="https://twitter.com/"><img src="./icons/Social_icon1.png"
+                                alt="Twitter"></a></li>
+                    <li class="social-media"><a href="https://www.linkedin.com/"><img src="./icons/Social_icon2.png"
+                                alt="Linkedin"></a></li>
+                    <li class="social-media"><a href="https://medium.com/"><img src="./icons/Social_icon3.png"
+                                alt="Medium"></a></li>
+                    <li class="social-media"><a href="https://angel.co/"><img src="./icons/Social_icon5.png"
+                                alt="angellist"></a></li>
+                </ul>
             </section>
         </div>
     </div>
+    <div class="container2">
+        <a href="#main-heading"><img class="arrow" src="./icons/down-arrow.svg" alt="Arrow"></a>
+        <h2 id="main-heading">My Recent <br>WORKS</h2>
+        <img id="indicator" src="./icons/Indicator.svg" alt="indicator">
+    </div>
+
+    <div class="card-holder">
+
+        <div class="card-holder-item">
+
+            <div class="holder-image">
+
+            </div>
+            <div class="holder-title">
+                <h5>Multi-Post Stories Gain+Glory</h5>
+            </div>
+            <div class="holder-discription">
+
+                <ul class="tag">
+
+                    <li class="tag-item" >
+                        Ruby on rails
+                    </li>
+                    <li class="tag-item" >
+                        Css
+                    </li>
+                    <li class="tag-item" >
+                        JavaScript
+                    </li>
+                    <li class="tag-item" >
+                        HTML
+                    </li>
+                </ul>
+
+                <button type="button" class="btn">See Project</button>
```

Comment:
I see your button has the shadow on hover, not like the Figma design please try to stick to the Figma design